### DISCUSSION TITLE
cannot code sign because the target does not have an info.plist file

### DISCUSSION THREAD

**Discussion by archan96** [invertase/react-native-firebase#6715] [Q&A]
Whenever trying to build after following the iOS setup instructions in react native firebase documentation in current react native version 0.70. Every package that I have installed giving me this error on Xcode "cannot code sign because the target does not have an info.plist file". Please help.

**Comment by mikehardy** [comment]
This script starts from scratch by creating a new project then integrates react-native-firebase and tests debug and release mode builds on android and iOS, it has no problem with codesigning as cross-checked by it's build of the macCatalyst target where codesigning is required.

So this is a reproducible example of it working: https://github.com/mikehardy/rnfbdemo/blob/main/make-demo.sh

If you have a failure of some sort, you'll need to produce a reproducible example of it not working https://stackoverflow.com/help/minimal-reproducible-example / https://stackoverflow.com/help/how-to-ask
